docker-compose_development_gdp.yml

```diff
@@ -327,6 +327,37 @@ services:
         target: /home/mig/state/mig_system_run
     command: /app/docker-entry.sh
 
+  migrid-lustre-quota:
```

Comment:
Hmmm, I'm not convinced we want Lustre in this 'toy' setup at all.
https://docker-migrid.readthedocs.io/en/latest/sections/getting-started/introduction.html#example-deployments

IMO, we should really consider a production_gdp example with realistic production settings for a full _SIF_-like deployment.